Makefile

```diff
@@ -64,16 +68,21 @@ fixtures/rpm-invalid-updateinfo:
 
 fixtures/rpm-unsigned:
 	rpm/gen-fixtures.sh $@ rpm/assets
-	rpm/del-fixtures-sign.sh $@
 
 fixtures/rpm-updated-updateinfo:
 	rpm/gen-patched-fixtures.sh $@ rpm/updated-updateinfo.patch
 
-fixtures/srpm:
+fixtures/srpm: gnupghome
 	rpm/gen-fixtures.sh $@ rpm/assets-srpm
+	GNUPGHOME=$$(realpath -e gnupghome) rpmsign --define '_gpg_name Pulp QE' \
+		--addsign --fskpath ./rpm/GPG-RPM-PRIVATE-KEY-pulp-qe --signfiles \
+		$$(find $@ -name '*.src.rpm')
```

Comment:
These three lines are almost copied and pasted from above. This is an area that's ripe for a future refactor.